src/contexts/UserDataContext.jsx

```diff
@@ -1,30 +1,21 @@
 // React Imports
 import React from 'react';
 // Context Imports
-import {
-  MessageContextProvider,
-  UserListContextProvider,
-  SignedInUserContextProvider,
-  DocumentListContextProvider
-} from '.';
+import { MessageContextProvider, DocumentListContextProvider } from '.';
 
 /**
  * A collection of React Contexts used for PASS to be used as a single wrapper
  *
  * @memberof contexts
  * @function UserDataContextProvider
- * @param {React.JSX.Element} children - The wrapped components that consumes
+ * @param {React.JSX.Element} children - The components that consume the data context
+ * @returns {React.JSX.Element} - A react app
  * Context from Provider
  */
-
 const UserDataContextProvider = ({ children }) => (
-  <SignedInUserContextProvider>
```

Comment:
I believe `SignedInUserContextProvider` is still in use? It's just `UserListContextProvider` being replaced.

Reply:
fixed